draft-ietf-mls-protocol.md

```diff
@@ -748,6 +752,7 @@ enum {
 
 struct {
     opaque identity<0..2^16-1>;
+    SignatureScheme algorithm;
     SignaturePublicKey public_key;
 } BasicCredential;
```

Comment:
Should that just be `Credential` ? Otherwise `BasicCredential` should probably be used in "UserInitKey".

Reply:
`Credential` is the generic object (with a type selector).  `BasicCredential`is a specific type of credential that just lists the required info (without, say, a signature from a CA).